File: 프로그래머스/0/181906. 접두사인지 확인하기/접두사인지 확인하기.cs
using System;

public class Solution {
    public int solution(string my_string, string is_prefix) {
        int answer = 0;
        if(my_string.Length < is_prefix.Length)
        {
            answer = 0;
        }
        else
        {
            answer = 1;
            for(int i = 0; i < is_prefix.Length; i++)
            {
                if(is_prefix[i] != my_string[i])
                {
                    answer = 0;
                    break;
                }
            }
        }
        
        return answer;
    }
}

File: 프로그래머스/0/181906. 접두사인지 확인하기/접두사인지 확인하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string my_string, string is_prefix) {
    for(int i = 1; i< my_string.size(); i++)
    {
        string temp = my_string.substr(0,i);
        if(temp == is_prefix)
            return 1;
    }
    return 0;
}